src/usr/local/www/status_logs_common.inc

```diff
@@ -0,0 +1,913 @@
+		$group->add(new Form_Checkbox(
+			'actpass',
+			'Pass',
+			'Pass',
+			in_arrayi('Pass', $Include_Act),
+			'Pass'
+		))->setWidth(1);
+
+		$group->add(new Form_Input(
+			'filterlogentries_time',
+			null,
+			'text',
+			$filterfieldsarray['time']
+		))->setWidth(3)->setHelp('Time');
+
+		$group->add(new Form_Input(
+			'filterlogentries_sourceport',
+			null,
+			'text',
+			$filterfieldsarray['srcport']
+		))->setWidth(2)->setHelp('Source Port');
+
+		$group->add(new Form_Input(
+			'filterlogentries_protocol',
+			null,
+			'text',
+			$filterfieldsarray['proto']
+		))->setWidth(2)->setHelp('Protocol');
+
+		$group->add(new Form_Input(
+			'filterlogentries_qty',
+			null,
+			'number',
+			$filterlogentries_qty,
+			['placeholder' => $nentries]
+		))->setWidth(2)->setHelp('Quantity');
+
+		$section->add($group);
+
+		$group = new Form_Group('');
+
+		$group->add(new Form_Checkbox(
+			'actblock',
+			'Block',
+			'Block',
+			in_arrayi('Block', $Include_Act),
+			'Block'
+		))->setWidth(1);
+
+		$group->add(new Form_Input(
+			'filterlogentries_interfaces',
+			null,
+			'text',
+			$filterfieldsarray['interface']
+		))->setWidth(3)->setHelp('Interface');
+
+		$group->add(new Form_Input(
+			'filterlogentries_destinationport',
+			null,
+			'text',
+			$filterfieldsarray['dstport']
+		))->setWidth(2)->setHelp('Destination Port');
+
+		$group->add(new Form_Input(
+			'filterlogentries_protocolflags',
+			null,
+			'text',
+			$filterfieldsarray['tcpflags']
+		))->setWidth(2)->setHelp('Protocol Flags');
+
+		$btnsubmit = new Form_Button(
+			'filterlogentries_submit',
+			' ' . gettext('Apply Filter'),
+			null,
+			'fa-filter'
+		);
+	} else { // Simple log filter form
+		$form = new Form(false);
+
+		$section = new Form_Section('Log Filter', 'basic-filter-panel', COLLAPSIBLE|$filter_state);
+
+		$group = new Form_Group('');
+
+		$group->add(new Form_Select(
+			'interface',
+			'Interface',
+			$interfacefilter,
+			build_if_list()
+		))->setWidth(2)->setHelp('Interface');
+
+		$group->add(new Form_Input(
+			'filterlogentries_qty',
+			null,
+			'number',
+			$filterlogentries_qty,
+			['placeholder' => $nentries]
+		))->setWidth(2)->setHelp('Quantity');
+
+		$section->add($group);
+
+		$group = new Form_Group('');
+
+		$group->add(new Form_Input(
+			'filtertext',
+			null,
+			'text',
+			$filtertext
+		))->setWidth(6)->setHelp('Filter Expression');
+
+		$btnsubmit = new Form_Button(
+			'filtersubmit',
+			' ' . gettext('Apply Filter'),
+			null,
+			'fa-filter'
+		);
+	}
+
+	$btnsubmit->removeClass('btn-primary')->addClass('btn-success')->addClass('btn-sm');
+
+	$group->add(new Form_StaticText(
+		'',
+		$btnsubmit
+	));
+
+	$group->setHelp('<a target="_blank" href="http://www.php.net/manual/en/book.pcre.php">' . gettext('Regular expression reference') . '</a> ' . gettext('Precede with exclamation (!) to exclude match.'));
+	$section->add($group);
+	$form->add($section);
+	print($form);
+}
+?>
+
+<?php
+function build_if_list() {
```

Comment:
I see that this function had this name already in the old code. I am a bit surprised that a function like this does not already exist, but that is another thing that could be looked into for the future (rationalizing all these sorts of places where this stuff is done).
For now I am just thinking that it could have a more specific name, as build_if_list() is likely to be chosen as a function name one day in interfaces.inc .or util.inc or...
Maybe status_logs_build_if_list?